Pokégen.PkHex/Services/PokemonService.cs

```diff
@@ -40,15 +42,11 @@ public async Task<PKM> GetPokemonFromFormFileAsync(IFormFile file, SupportedGame
 	public Task<PKM> GetPokemonFromShowdown(string showdownSet, SupportedGame game)
 	{
 		var set = new ShowdownSet(showdownSet);
-		var template = set.GetTemplate();
-
-		var invalidLines = set.InvalidLines;
-
-		if (invalidLines.Count != 0)
-			return Task.FromException<PKM>(new ShowdownException($"Unable to parse Showdown Set:\n{string.Join("\n", invalidLines)}"));
+		var template = new RegenTemplate(set);
 
 		var sav = game switch
 		{
+			SupportedGame.LGPE => SaveUtil.GetBlankSAV(GameVersion.GE, "SysBot.py"),
```

Comment:
Please do the same as i did for the others with `AutoLegalityModService.GetTrainerInfo` for the specific type